test/test_operations.py

```diff
@@ -1649,6 +1649,40 @@ def test_cached_addcdiv(self):
     xm.mark_step()
     self.assertEqual(met.metric_data("TransferToServerTime")[0], 4)
 
+  def test_print_executation(self):
+    xla_device = xm.xla_device()
+    xm.mark_step()
+    met.clear_all()
+
+    # case 1 mark_step
+    t1 = torch.randn(1, 4, device=xla_device)
+    xm.mark_step()
+    xm.wait_device_ops()
+    self.assertEqual(met.metric_data('ExecuteTime')[0], 1)
```

Comment:
side note: we need a less obscure way to get metrics. I assume `met.metric_data(...)[0]` is the number of samples for a metric?

Reply:
+1 for this, I think it should be pretty simple, we probably just need to add some getter functions. Yeah currently 0th index is `TotalSamples`. Ex. output looks like:
```
Metric: OutboundData
  TotalSamples: 3
  Accumulator: 12.00B
  ValueRate: 34.00B / second
  Rate: 8.50056 / second
  Percentiles: 1%=4.00B; 5%=4.00B; 10%=4.00B; 20%=4.00B; 50%=4.00B; 80%=4.00B; 90%=4.00B; 95%=4.00B; 99%=4.00B
  ```

Reply:
yea sg

Reply:
Not blocking though, I actually had a local branch that somewhat does this l when I was debugging something else with metrics -- so I can probably follow-up with a PR.